src/Orleans.Core/Runtime/CallbackData.cs

```diff
@@ -1,10 +1,24 @@
 using System;
+using System.Diagnostics.Tracing;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using Orleans.Transactions;
 
 namespace Orleans.Runtime
 {
+    public class OrleansCallBackDataEvent : EventSource
```

Comment:
These classes should perhaps go in a separate file, maybe on that's specific for event sources

Reply:
go to the same file in core abstraction? Or each project holds its own file.